cql3/statements/create_table_statement.cc

```diff
@@ -426,6 +427,7 @@ void create_table_statement::raw_statement::add_column_alias(::shared_ptr<column
 // in the table's options are done elsewhere.
 std::optional<sstring> check_restricted_table_properties(
     query_processor& qp,
+    std::optional<schema_ptr> schema,
     const sstring& keyspace, const sstring& table,
```

Comment:
we need to move this change (which adds optional schema in function declaration) to the previous patch. otherwise, we break bisectability.

Reply:
Right, `rebase`'ing weirdos. :roll_eyes:

Reply:
All set, flagged as resolved.